recipes/replit/0001-add-script.patch

```diff
@@ -0,0 +1,12 @@
+--- pyproject.toml	2021-07-12 20:10:09.540189500 -0400
++++ pyproject.toml.new	2021-07-12 20:10:42.447693300 -0400
```

Comment:
yeah, for completeness, i guess remove the `.new`

Reply:
got it, I'll remove it